packages/layout/src/FlexGrid.ts

```diff
@@ -0,0 +1,139 @@
+import { HTMLWidget } from "@hpcc-js/common";
+
+import "../src/FlexGrid.css";
+
+export class FlexGrid extends HTMLWidget {
+    constructor() {
+        super();
+    }
+    enter(domNode, element) {
+        super.enter(domNode, element);
+    }
+    update(domNode, element) {
+        super.update(domNode, element);
+        const context = this;
+        this.resize();
```

Comment:
Is this needed?  Our typical mantra is to put the onus on the hosting app to call `resize().render` if/when the hosting DIV changes size?